org.civicrm.bootstrapcivicrm/bootstrapcivicrm.php

```diff
@@ -123,22 +123,27 @@ function bootstrapcivicrm_civicrm_alterSettingsFolders(&$metaDataFolders = NULL)
 }
 
 /**
- * Implementation of hook_civicrm_pageRun
+ * Implementation of hook_civicrm_coreResourceList
  */
-function bootstrapcivicrm_civicrm_pageRun($page) {
-  if (!(isset($_GET['snippet']) && $_GET['snippet'] == 'json')) {
-    CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/bootstrap.css');
+function bootstrapcivicrm_civicrm_coreResourceList(&$items, $region) {
```

Comment:
I updated the commit message to explain. Basically, `hook_civicrm_coreResourceList` should be used for adding resources (css/js) which is exactly what we're doing here.